draft-terbu-sd-jwt-vc.md

```diff
@@ -31,8 +31,9 @@ organization="Authlete Inc. "
 .# Abstract
 
 This specification describes data formats as well as validation and processing
-rules to express Verifiable Credentials with JSON payload based on the SD-JWT
-format [@!I-D.ietf-oauth-selective-disclosure-jwt].
+rules to express Verifiable Credentials with a JWT Claim set [@!RFC7519] that are protected
+using the JWS [@!RFC7515], or the SD-JWT [@!I-D.ietf-oauth-selective-disclosure-jwt]
```

Comment:
```suggestion
rules to express Verifiable Credentials as a JWT [@!RFC7519] or an SD-JWT [@!I-D.ietf-oauth-selective-disclosure-jwt]
```

proposed simplification

Reply:
I think we need to highlight that we are proposing two different ways to protect JSON payload (JWT claim set) with and without SD, and the current text is more explicit about it.

Reply:
I think the suggested text from @tplooker conveys it too.

Reply:
as you said yourself, Brian, `a JWS with a JWT claim set is a JWT`, which is why I am ok calling it a JWT VC, but at least in the intro I think we agreed to make it clear that `a JWS with a JWT claim set is a JWT`

Reply:
I think Tobias' proposal is just fine.

Reply:
```suggestion
rules to express Verifiable Credentials based on a JWT [@!RFC7519] or an SD-JWT [@!I-D.ietf-oauth-selective-disclosure-jwt]
```